packages/types/src/schemas/i18n.ts

```diff
@@ -56,6 +56,13 @@ export const i18nSchema = z.object({
    */
   prepareEnvironmentTitleText: z.string().optional().describe('Text shown on top of the steps section.'),
 
+  /**
+   * Text shown on top of the preview section.
+   *
+   * @default 'Preview'
+   */
+  previewTitleText: z.string().optional().describe('Text shown on top of the preview section.'),
```

Comment:
I would be happy to merge this PR if we rename that text to `defaultPreviewTitleText`. What do you think @AriPerkkio ?

Reply:
I agree, `defaultPreviewTitleText` works better here. Let's use it.

Reply:
done